From 548aaecdf1a8b3801404ea16a1ad04251043b102 Mon Sep 17 00:00:00 2001
From: Tomi Valkeinen <tomi.valkeinen@ti.com>
Date: Tue, 10 Dec 2013 12:24:25 +0200
Subject: [PATCH 08/31] Revert "ARM: dts: omap3-igep0020: name twl4030 VPLL2
 regulator as vdds_dsi"

As we're adding proper DT support, this hack is no longer needed.

This reverts commit 2f2befd820252d05be7b1f5528a57af85338f59c.
---
 arch/arm/boot/dts/omap3-igep0020.dts | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/arch/arm/boot/dts/omap3-igep0020.dts b/arch/arm/boot/dts/omap3-igep0020.dts
index 1c7e74d..b9a9e17 100644
--- a/arch/arm/boot/dts/omap3-igep0020.dts
+++ b/arch/arm/boot/dts/omap3-igep0020.dts
@@ -215,8 +215,3 @@
 &usbhsehci {
 	phys = <&hsusb1_phy>;
 };
-
-&vpll2 {
-        /* Needed for DSS */
-        regulator-name = "vdds_dsi";
-};
-- 
1.8.5.1

